### ISSUE TITLE
Please link to sentence collector on voice webpage

### ISSUE THREAD

**Issue by Bullnados** [common-voice/common-voice#1887]
#1858 
Perhaps this closed issue is already the same?

But I'm missing a link from voice.mozilla.org to https://common-voice.github.io/sentence-collector/#/

**Comment by Gregoor** [comment]
So we are linking to it from the FAQ now. Where do you think the link should be placed?

**Comment by MichaelNMaggs** [comment]
For most readers that's a pretty obscure link on an obscure page!  As a lack of sentences has been causing problems in several languages, and still is, I'd suggest you need a fairly prominent link on the main page https://voice.mozilla.org/en:   Read/Listen/Upload sentences are all important.

**Comment by Bullnados** [comment]
AS Michael said.
Direct on main page,

**Comment by Nihisil** [comment]
I agree. I found the link to https://common-voice.github.io/ in closed PR which is not a really smooth way of finding things. Add this link to the main page and to the not completed language page will be a helpful solution.

**Comment by davidak** [comment]
@Gregoor can you prioritize this? Should not be too much work but would have a big impact.

**Comment by davidak** [comment]
This is still a huge issue!

**Comment by moz-bozden** [comment]
The sentence-collector has been integrated into the main common-voice workflows.
Closing this.
